source/Core/Src/settingsGUI.cpp

```diff
@@ -300,6 +300,10 @@ const menuitem solderingMenu[] = {
   {SETTINGS_DESC(SettingsItemIndex::TempChangeLongStep), nullptr, displayTempChangeLongStep, nullptr, SettingsOptions::TempChangeLongStep, SettingsItemIndex::TempChangeLongStep, 6},
   /* Locking Mode */
   {SETTINGS_DESC(SettingsItemIndex::LockingMode), nullptr, displayLockingMode, nullptr, SettingsOptions::LockingMode, SettingsItemIndex::LockingMode, 7},
+#ifdef TIP_TYPE_SUPPORT
+  /* Tip Type */
+  {SETTINGS_DESC(SettingsItemIndex::SolderingTipType), nullptr, displaySolderingTipType, showSolderingTipType, SettingsOptions::SolderingTipType, SettingsItemIndex::SolderingTipType, 5},
+#endif /* TIP_TYPE_SUPPORT */
```

Comment:
I took liberty to move the setting down inside of the soldering menu since in the UX design & usability usually they put variable/non-constant options closer to "the end" of "the list". So while temperature settings are applied to nearly every target (hence, "first in a row"), targets with support of different tips are limited. Just for the same reason we don't put "[thermo] profiles" submenu in the top of the list.